src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -324,11 +324,6 @@ private void testRemindActions() {
     }
 
     private void testPublishUnpublishActions() {
-        ______TS("publish action: AWAITING feedback session");
-
-        homePage.verifyUnclickable(homePage.getPublishLink(feedbackSessionAwaiting.getCourseId(),
-                                                           feedbackSessionAwaiting.getFeedbackSessionName()));
```

Comment:
Reverting this change because of [this](https://github.com/TEAMMATES/teammates/pull/7312#discussion_r124199216) suggestion and adding a test for caret button too.